lectures/debugging.md

```diff
@@ -255,3 +260,300 @@ There are many other useful magics:
 
 The full list of magics is [here](http://ipython.readthedocs.org/en/stable/interactive/magics.html).
 
+
+## Handling Errors
+
+```{index} single: Python; Handling Errors
+```
+
+Errors are problems within a program that stops the program from running further.
```

Comment:
"stops" -> "stop"

Is this true?  If they are caught in a try-except block, then execution continues?